### PR TITLE
Feature/dropdown component

### PR DESCRIPTION
**Pull request by weberjavi**
This PR creates dropdown component.   
A working example could be seen at the home page.  
image

### CODE REVIEW

File: app/javascript/app/components/dropdown/dropdown-component.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import styles from './dropdown-styles.scss';
+
+class Dropdown extends PureComponent {
+  componentDidUpdate() {
+    this.selectorElement.highlightFirstSelectableOption();
+  }
+  render() {
+    const {
+      label,
+      white,
```
Comment (Bluesmile82) [inline app/javascript/app/components/dropdown/dropdown-component.jsx]: Please remove all this props white, transparent, plain, ... as we are not going to use them and update the docs. If we want to use variations is better to use themes

File: app/javascript/app/components/dropdown/dropdown-styles.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+.dropdownWrapper {
+  position: relative;
+  width: 100%;
+
+  &.flex {
+    display: flex;
+  }
+}
+
+.dot {
```
Comment (Bluesmile82) [inline app/javascript/app/components/dropdown/dropdown-styles.scss]: I think we are not using the dot in this app

File: app/javascript/app/components/dropdown/dropdown-styles.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  font-weight: $font-weight-bold;
+  color: $theme-color;
+  margin-bottom: 4px;
+}
+
+.noScrollParent {
+  overscroll-behavior: contain; /* stylelint-disable-line property-no-unknown */
+}
+
+.withDot {
```
Comment (Bluesmile82) [inline app/javascript/app/components/dropdown/dropdown-styles.scss]: Not used, remove it, please

**Review by Bluesmile82 — CHANGES_REQUESTED** [review]
Please review the comments and remove unused props and styles. The rest is cool!

**Comment by weberjavi** [comment]
@Bluesmile82 feel free to review the changes I've made, thanks!

**Review by Bluesmile82 — APPROVED** [review]

**Comment by Bluesmile82** [comment]
Thanks! Much cleaner

**Pull request merged**